js/scripts.js

```diff
@@ -54,4 +54,68 @@ var TxtType = function(el, toRotate, period) {
         css.type = "text/css";
         css.innerHTML = ".typewrite > .wrap { border-right: 0.08em solid #fff}";
         document.body.appendChild(css);
-    };
\ No newline at end of file
+    };
+
+
+
+//API connection
+const contentDiv = document.getElementById('content');
+const api_url = fetch('https://www.edeka.de/eh/service/eh/offers');
```

Comment:
this is not an api url, it's a function)
I think you wanted to have something like:
`const apiUrl = 'https://www.edeka.de/eh/service/eh/offers';`